content/shmem_team_split_strided.tex

```diff
@@ -59,6 +58,18 @@
 relative order with respect to the parent team.
 If a $stride$ value equal to 0 is passed to \FUNC{shmem\_team\_split\_strided},
 then the $size$ argument passed must be 1, or the behavior is undefined.
+If the triplet provided to \FUNC{shmem\_team\_split\_strided} implies a
+wrap-around sequence, the input is considered invalid and the behavior is
+undefined.
+In other words, when $stride$ is nonzero, a newly created team must only
+include \acp{PE} whose subsequent parent \ac{PE} values are either all
+increasing (for positive $stride$) or all decreasing (for negative
+$stride$).
+That is, \textit{wrap-around} with respect to the parent team's \ac{PE} values
+is not permitted.
+For example, the list of \acp{PE} in the parent team should not start at a high
+number and then continue to include \acp{PE} in the lower end of the parent
+team's \ac{PE} range.
```

Comment:
@wokuno To change to "Given a parent team with XYZ PEs, a call to shmem_team_split strided with the following arguments would be invalid."